### ISSUE TITLE
Update raw-window-handle to 0.3

### ISSUE THREAD

**Issue by jazzfool** [gfx-rs/wgpu-rs#95]
The current version of the `raw-window-handle` dependency is `0.1`, which is missing some crucial things (for example `RawWindowHandle` can't be copied or cloned). This is already fixed for wgpu by https://github.com/gfx-rs/wgpu/pull/344.

**Comment by parasyte** [comment]
I was able to get this working just by updating `wgpu-native` and `winit`. Both dependencies have recently updated to `raw-window-handle` 0.3, but neither have been published yet.

```patch
diff --git a/Cargo.toml b/Cargo.toml
index ce82ea5..3f6ab65 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,9 +23,9 @@ default = []
 vulkan = ["wgn/gfx-backend-vulkan"]

 [dependencies]
-wgn = { package = "wgpu-native", git = "https://github.com/gfx-rs/wgpu", rev = "c3609d71c95dc5be1672c737ee4010e6a9408748" }
+wgn = { package = "wgpu-native", git = "https://github.com/gfx-rs/wgpu", rev = "5ba923c79fcd7e3e3a53c5fc7d64cf169c01050d" }
 arrayvec = "0.4"
-raw-window-handle = "0.1"
+raw-window-handle = "0.3"
 zerocopy = "0.2"

 [dev-dependencies]
@@ -34,4 +34,4 @@ env_logger = "0.6"
 glsl-to-spirv = "0.1"
 log = "0.4"
 png = "0.15"
-winit = "0.20.0-alpha3"
+winit = { git = "https://github.com/rust-windowing/winit", rev = "34dce8069f2a0606fdadf6b4cac4a7e059a9e9a2" }
```

**Comment by parasyte** [comment]
@kvark This is fixed by #99
